app/posts/register-trainee-teachers/2020-10-26-beta-prototype-updates.md

```diff
@@ -0,0 +1,217 @@
+      img: { src: "2020.11.02_12_19_41_Add a trainee - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Contact details",
+      img: { src: "2020.11.02_12_19_53_Contact details - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Contact details",
+      img: { src: "2020.11.02_12_20_01_Contact details - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Contact details",
+      img: { src: "2020.11.02_12_20_10_Contact details - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Confirm contact details",
+      img: { src: "2020.11.02_12_21_48_Confirm contact details - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add a trainee",
+      img: { src: "2020.11.02_12_21_58_Add a trainee - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Has the trainee shared diversity information?",
+      img: { src: "2020.11.02_12_22_07_Has the trainee shared diversity information_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Confirm diversity information",
+      img: { src: "2020.11.02_12_22_17_Confirm diversity information - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "What is their ethnic group?",
+      img: { src: "2020.11.02_12_22_28_What is their ethnic group_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Which of the following best describes their White background?",
+      img: { src: "2020.11.02_12_22_35_Which of the following best describes their White background_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Which of the following best describes their White background?",
+      img: { src: "2020.11.02_12_22_44_Which of the following best describes their White background_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Has the trainee shared that they are disabled?",
+      img: { src: "2020.11.02_12_23_25_Has the trainee shared that they are disabled_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Which disabilities were shared?",
+      img: { src: "2020.11.02_12_23_35_Which disabilities were shared_ - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Confirm diversity information",
+      img: { src: "2020.11.02_12_23_48_Confirm diversity information - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add a trainee",
+      img: { src: "2020.11.02_12_25_02_Add a trainee - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_11_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_18_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
+    }, {
+      text: "Add undergraduate degree",
+      img: { src: "2020.11.02_12_25_26_Add undergraduate degree - Register trainee teachers - GOV.UK.png" }
```

Comment:
The title of each screenshot should be unique

Reply:
Sorted.